docs/interfaces/c61/usb.md

```diff
@@ -0,0 +1,13 @@
+---
+title: USB (C61)
+tags:
+  - USB
+  - C61
+---
+## Overview
+* One USB-A 2.0 with OTG support. 
+
+* The currently provided BSP (using Linux 4.19) does not support the OTG functionality. Instead, the USB port is just a USB Host port. This will be fixed in an coming BSP release. If you are using Linux 4.4 you will be able to use 'Ethernet over USB'.*
```

Comment:
```suggestion
* The currently provided BSP (using Linux 4.19) does not support the OTG functionality. Instead, the USB port is just a USB Host port. This will be fixed in a coming BSP release. If you are using Linux 4.4 you will be able to use 'Ethernet over USB'.*
```